### PR TITLE
Support feedback Week 23

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

The improvements and fixes proposed here follow feedback by our support team.

## Proposal

- [x] Change image format on blog post and course glimpses to 300x170,
- [x] Allow only formatted text and image plugins in the course complementary information section,
- [x] Simplify permissions required to create Richie extension pages (we used to require permissions on the current page but it does not really make sense. We can just require the right to create a course, a blog post, etc.)
- [x] Show course creation wizard only on organization pages (it makes sure that the course being created is automatically linked to the current organization and that the organization's admin user group gets automatic access to the course),
- [x] Fix permission on courses filer folders for organization admin user groups,
- [x] Add a test to make sure that the `complementary information` section on courses is hidden when empty.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,17 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+- Allow only formatted texts or images in course complementary information,
+- Simplify why permissions are required to create Richie extension pages,
+- Show course creation wizard only on organization pages.
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply CHANGELOG.md]: I find it clearer and cleaner since some stretch on several lines... but I'm open to anything.
